src/commands/node/handlers.ts

```diff
@@ -552,6 +570,7 @@ export class NodeCommandHandlers {
 
     const action = helpers.commandActionBuilder([
       this.tasks.initialize(argv, keysConfigBuilder.bind(this), null),
+      RemoteConfigTasks.loadRemoteConfig.bind(this)(argv),
```

Comment:
`solo node keys` is non mutating it has no need for RemoteConfig or LeaseManager.  It just creates keys on the local users file system for use later.  It can be ran at anytime.